test/data/cmumps2fhir-patient7.shex

```diff
@@ -0,0 +1,587 @@
+﻿PREFIX fhir:<http://hl7.org/fhir/> 
```

Comment:
It can wait.  The schema was originally auto-generated by the FHIR build process, right?